src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  & &__star:not(:last-child) {
```

Comment:
The selector `& &__star:not(:last-child)` seems to have an extra `&`. It should likely be `&__star:not(:last-child)` to apply the margin to all stars except the last one within a `.stars` container.